client/src/components/edit-client-page/index.jsx

```diff
@@ -0,0 +1,198 @@
+import * as React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import EditIcon from "@mui/icons-material/Edit";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import MuiPhoneNumber from "material-ui-phone-number";
```

Comment:
Hello, so I fixed the imask issue and made the changes for owner/creator card. I still kept the table for the client card while ensuring that the divider extended until the end of the card length. I did play around with Grid and Divider but couldn't really get it to work as needed. Therefore, for the sake of time, I decided to keep the table. Lastly, I know you mentioned that the Icon color schemes were a little off. Since I lost access to figma, I couldn't fix it but I will once I get the access. Thank you!